src/themes/index.js

```diff
@@ -6,11 +6,15 @@ LICENSE file in the root directory of this source tree.
 */
 // @flow
 
-import {default as LightThemeCreator} from './light-theme';
+import createTheme from './creator';
+import {primitives as lightThemePrimitives} from './light-theme';
 
-export const LightTheme = LightThemeCreator();
+export const LightTheme = createTheme({
+  ...lightThemePrimitives,
+});
```

Comment:
Probably don't need the spread here since we're just using `lightThemePrimitives` directly and createTheme is pure.